app/styles/_toolbar.scss

```diff
@@ -69,14 +69,18 @@
       margin: 1em 0;
     }
   }
-
+  color: #fff;
   font-size: $font-size;
-  background: $ember-orange top center;
-  background-image: url("../images/navigation_background.png");
-  border-bottom: 1px solid $burnt-orange;
-  -moz-box-shadow: rgba(0,0,0,0.3) 0 0 3px;
-  -webkit-box-shadow: rgba(0,0,0,0.3) 0 0 3px;
-  box-shadow: rgba(0,0,0,0.3) 0 0 3px;
+  background: $ember-orange;
+  background-image: url("//emberjs.com/images/header.svg");
```

Comment:
Please download and include this image in build. This will cause a mixed content warning in production as emberjs.com does not use https.